homepage.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import (
+"net/http"
+)
+
+func startUiMode() (int) {
+	mux := http.NewServeMux()
+	mux.Handle("/", http.FileServer(http.Dir("./ui")))
+	http.ListenAndServe(":9191", mux)
+	return http.StatusOK
```

Comment:
no need to return statusOK. file server should serve the appropriate file with http status 200